pom.xml

```diff
@@ -1,1076 +1,1076 @@
+    <name>LM SL CF Parent</name>
+    <description>Multi-Target Application (MTA) deployment service for Cloud Foundry</description>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <junit.version>5.3.1</junit.version>
+        <junit.platform.version>1.3.1</junit.platform.version>
+        <hamcrest.version>1.3</hamcrest.version>
+        <sonar.core.codeCoveragePlugin>jacoco</sonar.core.codeCoveragePlugin>
+        <sonar.dynamicAnalysis>reuseReports</sonar.dynamicAnalysis>
+        <sonar.jacoco.reportPath>target/jacoco.exec</sonar.jacoco.reportPath>
+        <mockito.version>1.9.5</mockito.version>
+        <derby.version>10.12.1.1</derby.version>
+        <hsqldb.version>2.3.2</hsqldb.version>
+        <servlet.api.version>4.0.0</servlet.api.version>
+        <jax-rs.version>2.0.1</jax-rs.version>
+        <javax.inject.version>1</javax.inject.version>
+        <log4j.version>1.2.17</log4j.version>
+        <slf4j.version>1.7.25</slf4j.version>
+        <commons-io.version>2.6</commons-io.version>
+        <commons-fileupload.version>1.3.3</commons-fileupload.version>
+        <commons-logging.version>1.2</commons-logging.version>
+        <commons-lang3.version>3.8.1</commons-lang3.version>
+        <commons-collections4.version>4.2</commons-collections4.version>
+        <httpclient.version>4.5.6</httpclient.version>
+        <httpcore.version>4.4.10</httpcore.version>
+        <jersey.version>2.23.2</jersey.version>
+        <hk2.version>2.4.0-b10</hk2.version>
+        <javassist.version>3.19.0-GA</javassist.version>
+        <javax.annotation-api.version>1.2</javax.annotation-api.version>
+        <validation-api.version>1.1.0.Final</validation-api.version>
+        <eclipselink.version>2.7.3</eclipselink.version>
+        <gson.version>2.6.2</gson.version>
+        <snakeyaml.version>1.23</snakeyaml.version>
+        <flowable.version>6.4.0</flowable.version>
+        <mybatis.version>3.3.0</mybatis.version>
+        <joda-time.version>2.8</joda-time.version>
+        <spring.version>5.1.2.RELEASE</spring.version>
+        <spring.security.version>5.1.5.RELEASE</spring.security.version>
+        <spring.security.oauth2.version>2.3.5.RELEASE</spring.security.oauth2.version>
+        <spring.security.jwt.version>1.0.9.RELEASE</spring.security.jwt.version>
+        <spring.cloud.version>1.2.3.RELEASE</spring.cloud.version>
+        <hana.cloud.version>1.0.4.RELEASE</hana.cloud.version>
+        <quartz.version>2.2.1</quartz.version>
+        <jackson.version>2.9.8</jackson.version>
+        <jackson-asl.version>1.9.13</jackson-asl.version>
+        <liquibase.version>3.4.1</liquibase.version>
+        <liquibase-slf4j.version>1.2.1</liquibase-slf4j.version>
+        <cloudfoundry.client.version>1.13.0-SNAPSHOT</cloudfoundry.client.version>
```

Comment:
When the PR is merged in the cf-java-client, release it and adopt the new version here, directly :)